modin/pandas/groupby.py

```diff
@@ -689,6 +710,11 @@ def _default_to_pandas(self, f, *args, **kwargs):
         else:
             by = self._by
 
+        # is_list_like() will be True for Categorical, which we don't want to
+        # convert to a list, so use isinstance(by, list):
+        if isinstance(by, list):
+            by = [o._to_pandas() if isinstance(o, Series) else o for o in by]
+
```

Comment:
`self._df._default_to_pandas` (which is below) already contains logic about converting its arguments to pandas if possible,  can we change `groupby_on_multiple_columns` that way, to it uses `by` not via closure but directly from obtained arguments? If yes, that we can remove those lines and `_default_to_pandas` automatically convert our `by` to pandas

Reply:
Done.